temperature/router.py

```diff
@@ -0,0 +1,35 @@
+from fastapi import APIRouter, Depends, Query, HTTPException
+from sqlalchemy.orm import Session
```

Comment:
The import of `Session` is not used in the code. Since you're using `AsyncSession` in your CRUD operations, ensure that the correct session type is used consistently across your application.